### PR TITLE
Upgrade qiniu

### PR DESCRIPTION
**Pull request by qiqizjl**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/qiniu/js-sdk/releases
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/qiniu-js/index.d.ts [hunk 1]
```diff
@@ -1,135 +1,145 @@
...
 // Project: https://github.com/qiniu/js-sdk#readme
 // Definitions by: taoqf <https://github.com/taoqf>
+//                 qiqizjl <https://github.com/qiqizjl>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.1
 
 export as namespace qiniu;
 
 export interface Next {
-	total: {
```
Comment (taoqf) [inline types/qiniu-js/index.d.ts]: Would you please use Tab?

Comment (taoqf) [reply types/qiniu-js/index.d.ts]: I could barely see the changes.

**Review by taoqf — CHANGES_REQUESTED** [review]

**Review by taoqf — APPROVED** [review]
Good job!

**Review by qiqizjl — COMMENTED** [review]
thx

**Comment by uniqueiniquity** [comment]
Congratulations on your first DefinitelyTyped contribution!
Thank you for being a part of the community!

**Pull request merged**